### DISCUSSION TITLE
The data repo has no data. There are cyclic references to how data can be downloaded but no clear and correct instructions. Certainly there are s3 and azure links but I have troubles to download data from there. Please make data available in the github repo

### DISCUSSION THREAD

**Discussion by snuup** [OvertureMaps/data#223] [Discussion]
### Category Feedback

Add details 
 - Expected behaviour you didn't see 
 - Unexpected behaviour 
 - suggested functionality 
 - Steps to reproduce the problem

**Comment by jwass** [comment]
Looks like this was made into an issue and closed in https://github.com/OvertureMaps/data/issues/224
